packages/tuono/src/build/config/create-json-config.ts

```diff
@@ -0,0 +1,48 @@
+import { promises as fs } from 'fs'
+import * as path from 'path'
```

Comment:
for import consistency across the codebase

```suggestion
import path from 'path'
```